suites/reef/rados/tier-3_rados_test-location-stretch-mode.yaml

```diff
@@ -322,7 +320,7 @@ tests:
         tiebreaker_mon_site_name: arbiter
         delete_pool: true
       desc: Test stretch Cluster netsplit scenario between data sites
-      comments: Active bug - 2249962
+      comments: Active bug - 2265116
```

Comment:
This is the latest active bug for the issue : https://bugzilla.redhat.com/show_bug.cgi?id=2318975